Marlin/src/pins/sanguino/pins_MELZI_TRONXY.h

```diff
@@ -26,6 +26,13 @@
  */
 
 #define BOARD_INFO_NAME "Melzi (Tronxy)"
+
+#if HAS_GRAPHICAL_LCD
+  #define BOARD_ST7920_DELAY_1 DELAY_NS(0)
+  #define BOARD_ST7920_DELAY_2 DELAY_NS(125)
+  #define BOARD_ST7920_DELAY_3 DELAY_NS(0)
+#endif
+
 #include "pins_MELZI.h"
```

Comment:
…Then we can move the overrides in front of the `#include` and everything should be happy.